packages/storybook/.storybook/main.js

```diff
@@ -15,10 +15,13 @@ module.exports = {
       name: '@storybook/addon-essentials',
       options: {
         actions: false,
-        controls: false
+        controls: false,
```

Comment:
How about add a url check here.
If current URL contains `/story/` we disable specific buttons and if it contains `/docs/` we disable some buttons.

Reply:
we can do that so are we saying on doc pages we keep it clean and on canvas pages we have all of the bells and whistles?

Reply:
No 100% sure about that, the original idea was to clean up the header so that canvas/doc button are more prominent.
Can we push all the buttons to the right?
Decluttering top left might also make the canvas/doc buttons prominent.

Reply:
It wasn't clear on how to move the positioning of the toolbar options to the right. I am in their discord right now trying to figure out how someone would do this. If the answer for now is to write my own toolbar to allow me to align it to the left or right then I will defer that for another time.